widget/embedded/src/components/RouteWarningsAndErrors/RouteWarningsAndErrors.helpers.tsx

```diff
@@ -0,0 +1,78 @@
+import type {
```

Comment:
Helpers should be pure JS and not JSX. Please make the function into smaller pieces to make this happens if that works.